addons/sys_sem70/CfgAcreRadios.hpp

```diff
@@ -0,0 +1,88 @@
+#include "script_component.hpp"
+
+class CfgAcreComponents
+{
+    class ACRE_BaseRadio;
+
+    class ACRE_SEM70_base : ACRE_BaseRadio {
+        class Interfaces;
+        isAcre = 1;
+    };
+
+    class ACRE_SEM70 : ACRE_SEM70_base {
+        name = "SEM 70";
+        sinadRating = -117;
+        sensitivityMin = -117;
+        sensitivityMax = -50;
+        isPackRadio = 1;
+        isDeployable = 0;
+
+        connectors[] =     {
```

Comment:
Again...this is in-line with the rest of acre. New PR would be the best solution